### DISCUSSION TITLE
MetricTracker

### DISCUSSION THREAD

**Discussion by huidiwiwi** [Lightning-AI/torchmetrics#1103] [Regression]
Hi, I am facing an issue using the metric tracker for validation of a regression model. It will be very helpful if someone could help me with an example use case. 
The metrics are error measurements, where the lower value is better. To avoid the tracker storing metrics from the validation step sanity checks, I reset the tracker under the `on_train_start()` hook in pytorch lightning, and initialize the tracker with `maximize=False `as well.
But the tracker increases weirdly during the training. the val/smape_best for example start from epoch 0 with 1 and raise to 2 and then 3 and hold until the end, despite the logged val/smape per epoch develop from 1.74 to 1.64
